main.go

```diff
@@ -61,6 +64,11 @@ func main() {
 	var leaderElectionNamespace string
 	var enableLeaderElection bool
 	var probeAddr string
+	var opts zap.Options
+
+	if strings.EqualFold(os.Getenv("VOL_REP_DEVELOPER"), "true") {
```

Comment:
`VOL_REP_DEVELOPER` to `DEVELOPMENT_MODE` or `RUN_AS_DEVELOPER`?

Reply:
It is usually recommended that you prefix the ENV with something specific to your binary/project. The recommendation is not as applicable to processes that mainly run in the containers where the processes are isolated in comparison to the processes that are generally run alongside many other processes.

Happy to do the change to DEVELOPMENT_MODE though.

Reply:
Done